scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -114,5 +114,6 @@ cargo-clippy:
     - .docker-env
     - .test-refs
   script:
+    - echo $RUSTFLAGS
     - cargo version && cargo clippy --version
-    - SKIP_WASM_BUILD=1 env -u RUSTFLAGS cargo clippy --locked --all-targets
+    - SKIP_WASM_BUILD=1 cargo +1.70 clippy --all-targets --locked -p polkadot-runtime-parachains
```

Comment:
```suggestion
    - rustup install 1.70 && SKIP_WASM_BUILD=1 cargo +1.70 clippy --all-targets --locked -p polkadot-runtime-parachains
```

Reply:
Obviously this is a shitty solution, but you now can debug from here why this is not working correctly in the docker without the `rustup`.